src/features/classification/security-controls/SecurityControlsModal.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../styles/variables';
+
+.bdl-SecurityControlsModal {
+    .link {
+        color: $bdl-box-blue !important;
```

Comment:
Do we really need to have the `!important` here?

Reply:
removed since we have removed the link element in the modal.